src/barretenberg/mod.rs

```diff
@@ -391,40 +383,8 @@ mod wasm {
         }
     }
 
-    fn clock_time_get(_: i32, _: i64, _: i32) -> i32 {
-        unimplemented!("clock_time_get is not implemented")
-    }
-
     fn proc_exit(_: i32) {
-        unimplemented!("proc_exit is not implemented")
```

Comment:
new barretenberg code provides wasi stubs for most methods not being used, so these can be removed